Procurement/ViewModel/ItemHoverViewModel.cs

```diff
@@ -44,6 +44,13 @@ public class ItemHoverViewModel
 
         public string ItemLevel { get; set; }
 
+        public bool IsGemProgressVisible => Item is Gem;
```

Comment:
What if the Json did not have valid experience information?  How would the progress bar be displayed then?

Reply:
This is what the fallback values are for in the ItemHover.xaml

If the item isn't a gem the whole section won't be visible.